suite-native/assets/src/components/useSelectedAssetSymbol.tsx

```diff
@@ -0,0 +1,14 @@
+import { atom, useAtom } from 'jotai';
+
+import { NetworkSymbol } from '@suite-common/wallet-config';
+
+const selectedAssetSymbolAtom = atom<NetworkSymbol | null>(null);
+
+export const useSelectedAssetSymbolAtom = () => {
+    const [selectedAssetSymbol, setSelectedAssetSymbol] = useAtom(selectedAssetSymbolAtom);
+
+    return Object.freeze({
```

Comment:
Why freeze?

Reply:
No need for it I guess, removed here
https://github.com/trezor/trezor-suite/pull/9937/commits/f8feb53881f1287c482ef65812db524b4acb805d